crates/core/src/config/mod.rs

```diff
@@ -3,7 +3,7 @@ use crate::error::FilesystemError;
 use crate::repository::CompressionFormat;
 use glob::MatchOptions;
 use serde::{Deserialize, Serialize};
-use std::collections::HashMap;
+use std::{collections::HashMap};
```

Comment:
Not sure how the `{  }` remained here. Can you remove or run `cargo fmt`?

Reply:
You are correct - Not sure why it added curly braces. Made a commit only for formatting.